Sources/CustomEvent/CustomEvent.swift

```diff
@@ -1,25 +1,20 @@
 //
 // CustomEvent.swift
 // Kommunicate
-//Created by Sathyan Elangovan on 09/12/21.
+// Created by Sathyan Elangovan on 09/12/21.
 
 import Foundation
 
 public enum CustomEvent: String {
     case messageSend = "ON_MESSAGE_SEND"
     case faqClick = "ON_FAQ_CLICK"
-    case notificationClick = "ON_NOTIFICATION_CLICK"
     case newConversation = "ON_START_NEW_CONVERSATION_CLICK"
-    case attachmentClick = "ON_ATTACHMENT_ICON_CLICK"
```

Comment:
What if someone is using these?

Reply:
once they upgrade to new version, these events will not considered.Wr have removed old callback & new callbacks has to be implemented .Will add the migration doc in release notes as well as in our docs

Reply:
we have another release in the next week, will add those events there